src/ac_lotus.c

```diff
@@ -0,0 +1,115 @@
+#include "ac_lotus.h"
+
+#include "m_name_table.h"
+#include "sys_matrix.h"
+#include "m_lib.h"
+#include "m_rcp.h"
+#include "m_common_data.h"
+#include "m_player_lib.h"
+
+#ifdef MUST_MATCH
+#ifndef __INTELLISENSE__
+/* Force assetrip to detect these assets. They're used in a .c_inc file. */
+FORCESTRIP static u16 __hack_pal0[] = {
+  #include "assets/aLOT_obj_01_lotus_pal.inc"
+};
+FORCESTRIP static u16 __hack_pal1[] = {
+  #include "assets/obj_02_lotus_pal.inc"
+};
+FORCESTRIP static u16 __hack_pal3[] = {
+  #include "assets/obj_03_lotus_pal.inc"
+};
+FORCESTRIP static u16 __hack_pal4[] = {
+  #include "assets/obj_04_lotus_pal.inc"
+};
+FORCESTRIP static u16 __hack_pal5[] = {
+  #include "assets/obj_05_lotus_pal.inc"
+};
+FORCESTRIP static u16 __hack_pal6[] = {
+  #include "assets/obj_06_lotus_pal.inc"
+};
+FORCESTRIP static u16 __hack_pal7[] = {
+  #include "assets/obj_07_lotus_pal.inc"
+};
+FORCESTRIP static u16 __hack_pal8[] = {
+  #include "assets/obj_08_lotus_pal.inc"
+};
+FORCESTRIP static u16 __hack_pal9[] = {
+  #include "assets/obj_09_lotus_pal.inc"
+};
+#endif
+#endif
+
+enum {
+  aLOT_ACTION_WAIT1,
+  aLOT_ACTION_WAIT2,
+  aLOT_ACTION_SHAKE,
+};
+
+static void aLOT_actor_ct(ACTOR* actor, GAME* game);
+static void aLOT_actor_dt(ACTOR* actor, GAME* game);
+static void aLOT_actor_init(ACTOR* actor, GAME* game);
+static void aLOT_actor_draw(ACTOR* actor, GAME* game);
+
+ACTOR_PROFILE Lotus_Profile = {
+  mAc_PROFILE_LOTUS,
+  ACTOR_PART_ITEM,
+  ACTOR_STATE_TA_SET,
+  LOTUS,
+  ACTOR_OBJ_BANK_KEEP,
+  sizeof(LOTUS_ACTOR),
+  &aLOT_actor_ct,
+  &aLOT_actor_dt,
+  &aLOT_actor_init,
+  &aLOT_actor_draw,
+  NULL
+};
+
+StatusData_c lotus_StatusData = { 0, 0, 0, 0, 0x0A};
+ClObjPipeData_c lotus_CoInfoData = {
+    {0x3D, 0x20, ClObj_TYPE_PIPE},  // collision data
+    {1},                            // element data
+    // Pipe specs
+    {
+        70, // radius
+        6,  // height
+        0,  // offset
+
+        {0, 0, 0},  // center
+    },
+};
+
+ClObjPipe_c pipeinfo;
```

Comment:
```suggestion
extern ClObjPipe_c pipeinfo;
```

Reply:
This is static, not external.
![image](https://github.com/Prakxo/ac-decomp/assets/24523422/d25b9e68-4206-41ba-8e64-ad6f2d2be839)

Reply:
Actually, it is external. It's only accessed locally though. Sorry.